### DISCUSSION TITLE
Is it possible to get the root HTMLElement object of a quasar component?

### DISCUSSION THREAD

**Discussion by luxi78** [quasarframework/quasar#13443] [General - Components / Directives / etc] [answered]
For example: if I wanna get the position and size of a QToolbar, I would prefer to try to obtain the root html element of this QTollbar obj,  then do the normal html+js things after that. Is it possible to do that?

**Comment by metalsadman** [comment, marked as answer]
use a template ref on the element, log its `$el` prop and walk into the object.

**Reply by luxi78** [reply]
AFAIK template ref could not refer to the root HTMLElement object of the vue component.

Please check this out:  [Sample Code](https://codesandbox.io/embed/cold-bash-h8lvso?fontsize=14&hidenavigation=1&theme=dark)
We can get parent div object using template ref, but couldn't get vue component's html element object

Any ideas?

**Reply by metalsadman** [reply]
`comp.value.$el`
![Screen Shot 2022-05-18 at 5 51 32 PM](https://user-images.githubusercontent.com/31038838/169011955-5cd242e1-62c4-4b69-9348-47c8e8725841.png)

**Reply by luxi78** [reply]
> `comp.value.$el` ![Screen Shot 2022-05-18 at 5 51 32 PM](https://user-images.githubusercontent.com/31038838/169011955-5cd242e1-62c4-4b69-9348-47c8e8725841.png)

Wow, It works! Thank you very much
